src/3.x/configuration.md

```diff
@@ -105,3 +105,19 @@ ELASTICSEARCH_SCHEME=
 ELASTICSEARCH_USER=
 ELASTICSEARCH_PASS=
 ```
+
+## Early hints
+
+We come with [Early Hints](https://github.com/justbetter/laravel-http3earlyhints) built in.
+You should enable your early hints in [Cloudflare](https://developers.cloudflare.com/cache/advanced-configuration/early-hints/#enable-early-hints) or your CDN for a boost
+
+Anything loaded with `@vite()` will properly get loaded with early hints, it will also discover other elements on your page which could benefit from early hints. Like your `preload`, and `preconnect` links, eager loaded images, blocking scripts and bocking styles.
```

Comment:
```suggestion
We come with [Early Hints](https://github.com/justbetter/laravel-http3earlyhints) built in. You should enable your early hints in [Cloudflare](https://developers.cloudflare.com/cache/advanced-configuration/early-hints/#enable-early-hints) or your CDN for a boost. Anything loaded with `@vite()` will properly get loaded with early hints, it will also discover other elements on your page which could benefit from early hints. Like your `preload`, and `preconnect` links, eager loaded images, blocking scripts and bocking styles.
```